code/core/package.json

```diff
@@ -271,7 +271,7 @@
   "dependencies": {
     "@storybook/csf": "^0.1.11",
     "@types/express": "^4.17.21",
-    "@types/node": "^18.0.0",
+    "@types/node": "^22.0.0",
```

Comment:
Is this a breaking change for users since it's a regular dependency?

Reply:
I'll broaden this version range, to be on the safe side. I'm convinced this won't be breaking.

I'll see if it can be moved to a devDep, actually.

Reply:
This change force-upgraded `@types/node` in our app repo to 22.x while we're using Node 20.x and don't want typings for a newer Node version. This effectively made Node 22.x the minimum required version of Node for Storybook with Webpack if you're using TypeScript.